packages/showcases/web-components/stories/components/vtmn-toggle/vtmn-toggle.stories.tsx

```diff
@@ -0,0 +1,59 @@
+import { html } from 'lit-html';
+import { spread } from '@open-wc/lit-helpers';
+
+export default {
+  title: 'Components/vtmn-toggle',
+  component: 'vtmn-toggle',
+  argTypes: {
+    id: {
+      type: { name: 'string', required: true },
+      description: 'The id of the toggle.',
+      defaultValue: 'my-id',
```

Comment:
For me it's required, because if not, we will have several toggles with the same id in a HTML page...